python/setup.py

```diff
@@ -78,43 +78,66 @@ def run(self):
         subprocess.check_call(["../build.sh", "-p", sys.executable])
 
         # We also need to install pyarrow along with Ray, so make sure that the
-        # relevant non-Python pyarrow files get copied.
-        pyarrow_files = []
+        # relevant pyarrow files get copied.
         for (root, dirs, filenames) in os.walk("./ray/pyarrow_files/pyarrow"):
-            for name in filenames:
-                pyarrow_files.append(os.path.join(root, name))
+            for filename in filenames:
+                path = os.path.join(root, filename)
+                self.copy_file(path, path)
 
-        files_to_include = ray_files + pyarrow_files
+        # Also copy the modin files.
```

Comment:
I think this fixes #3998

Reply:
yeah, I moved it to https://github.com/ray-project/ray/pull/4069